src/pages/Validators/DelegationValidatorsTable.tsx

```diff
@@ -56,11 +56,13 @@ function getValidatorsOrderedBy(
   column: Column,
 ) {
   switch (column) {
-    case "delegatedAmount":
```

Comment:
Why not keep this as well? :) Sorting by total delegation amount is useful

Reply:
`delegatedAmount` and other fields, like commission, delegator number, are either from view function or indexer query. i'm working on making all fields sortable but right now only column from json file's sortable, thats why only rewards earned can be sorted.

Reply:
I see